cmake/scripts/common/Macros.cmake

```diff
@@ -583,7 +583,8 @@ function(core_find_git_rev stamp)
   else()
     find_package(Git)
     if(GIT_FOUND AND EXISTS ${CMAKE_SOURCE_DIR}/.git)
-      execute_process(COMMAND ${GIT_EXECUTABLE} update-index --ignore-submodules --refresh -q)
+      execute_process(COMMAND ${GIT_EXECUTABLE} update-index --ignore-submodules --refresh -q
```

Comment:
please switch the order of the parameters `--refresh` and `-q`
For some reason git only is quite if `-q` is passed before `--refresh`.
My intention never was to get `... needs update` lines.